src/Explorer/Panes/PublishNotebookPaneAdapter.tsx

```diff
@@ -58,12 +62,30 @@ export class PublishNotebookPaneAdapter implements ReactAdapter {
     window.requestAnimationFrame(() => this.parameters(Date.now()));
   }
 
-  public open(
+  public async open(
     name: string,
     author: string,
     notebookContent: string | ImmutableNotebook,
-    parentDomElement: HTMLElement
-  ): void {
+    parentDomElement: HTMLElement,
+    isCodeOfConductEnabled: boolean
+  ): Promise<void> {
+    if (isCodeOfConductEnabled) {
+      try {
+        const response = await this.junoClient.isCodeOfConductAccepted();
+        if (response.status !== HttpStatusCodes.OK && response.status !== HttpStatusCodes.NoContent) {

```

Comment:
```suggestion
        if (response.ok) {

```

Reply:
this "response" is of type IJunoResponse, which only has the statusCode